src/librustc_codegen_llvm/debuginfo/metadata.rs

```diff
@@ -863,6 +865,21 @@ fn pointer_type_metadata(
     }
 }
 
+fn param_type_metadata(cx: &CodegenCx<'ll, 'tcx>, t: Ty<'tcx>) -> &'ll DIType {
+    debug!("param_type_metadata: {:?}", t);
+    let dl = cx.tcx.data_layout();
+    let name = SmallCStr::new(&format!("{:?}", t));
+    return unsafe {
+        llvm::LLVMRustDIBuilderCreateBasicType(
```

Comment:
Does DWARF not have something like "template parameter"? Oh I guess in C++ monomorphization is guaranteed, nvm.

Reply:
This was marked as resolved without a reply, what's the status here?

Reply:
I didn't do anything because you ended your question with "nevermind".

Reply:
Oh... Right.